.gitignore

```diff
@@ -1,3 +1,4 @@
+__pycache__/
```

Comment:
I don't think this is needed. We have this already for the `sdk/python` directory.
```suggestion
```